packages/api/test/models/activityLog.test.ts

```diff
@@ -89,4 +89,88 @@ describe("ActivityLog Model", function () {
             "The message attribute should have a maximum of 512 characters.",
         );
     });
+
+    it("should not be created when component is undefined", async () => {
+        const newLog = new ActivityLogModel({
+            ...logTemplate,
+            component: undefined,
+        });
+
+        await assertThrowsAsync(
+            async () => newLog.save(),
+            "The component attribute is required.",
+        );
+    });
+
+    it("should not be created when component is null", async () => {
+        const newLog = new ActivityLogModel({
+            ...logTemplate,
+            component: null,
+        });
+
+        await assertThrowsAsync(
+            async () => newLog.save(),
+            "The component attribute is required.",
+        );
+    });
+
+    it("should not be created when component is empty", async () => {
+        const newLog = new ActivityLogModel({
+            ...logTemplate,
+            component: "",
+        });
+
+        await assertThrowsAsync(
+            async () => newLog.save(),
+            "The component attribute is required.",
+        );
+    });
+
+    it("should not be created when component is invalid", async () => {
+        const newLog = new ActivityLogModel({
+            ...logTemplate,
+            component: "no-api",
+        });
+
+        await assertThrowsAsync(
+            async () => newLog.save(),
+            "The component attribute is not from defined origins.",
+        );
+    });
+
+    it("should run when context is null", async () => {
```

Comment:
What do you mean by "should run"? It should be "should be created". Follow the same convention everywhere.